include/deal.II/fe/fe_interface_values.h

```diff
@@ -126,6 +127,16 @@ namespace FEInterfaceViews
      * of the component selected by this view.
      */
     value_type
+    jump_in_values(const unsigned int interface_dof_index,
```

Comment:
Should this be `jump_in_shape_values()`, to be consistent with other places?

Reply:
I would say no. The views are value(), gradient (), etc. without "shape".

Reply:
@tjhei's got it ;-) This was deliberate, because I wanted to retain consistency with what we have elsewhere in the code.
- [FEValues::shape_value()](https://dealii.org/developer/doxygen/deal.II/classFEValuesBase.html#a1dd48cb744013c448d57f8f77640c08d)
- [FEValuesViews::Scalar::value()](https://dealii.org/developer/doxygen/deal.II/classFEValuesViews_1_1Scalar.html#a1049a4b0dafb4822a2e16595a15c004a)
- etc.

So I'd prefer to keep this as is, just so that we're consistent in what we do for FEValues/FEFaceValues/FEInterfaceValues vs their respective views. I feel that if we adjust the naming scheme in one place then we should adjust it in all places.